llmfoundry/data/finetuning/dataloader.py

```diff
@@ -280,6 +280,11 @@ def _validate_config(dataset_cfg: DictConfig) -> None:
                 'Using a streaming dataset requires setting both `remote` and `local`, ' +\
                 'but dataset.local is None.'
             )
+        else:
+            if dataset_cfg.get('split') not in os.listdir(dataset_cfg.local):
```

Comment:
I don't think this is exactly the right condition. If remote is not None, then local won't contain the split folder (yet).

Reply:
Good catch. Will address comments in a day or so. Thanks for the feedback